ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -49,6 +47,9 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+
+            if(!this.editMode) // Creating a new project.
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
I'm not really fan of this side effect :( Also as I said in some of previous comments, I would rather see privisional projects as strictly server thing which frontend doesn't really have to care about.

(I can see `do that after summit` comments already, so OK, I guess we can do that after summit, but I just want to point it out. And if we merge it as it is, we should create JIRA for similar issues as this one, so we don't forget about it).